tests/js/spec/views/organizationDashboard/projectCard.spec.jsx

```diff
@@ -94,7 +94,7 @@ describe('ProjectCard', function() {
   });
 
   it('renders with platform', function() {
-    expect(wrapper.find('Platformicon[platform="javascript"]')).toHaveLength(1);
+    expect(wrapper.find('PlatformList')).toHaveLength(1);
```

Comment:
We should make sure this renders the javascript icon

Reply:
Done! thank you 😄